ghost/core/core/server/data/migrations/versions/5.11/2022-08-23-13-41-backfill-members-created-events.js

```diff
@@ -0,0 +1,30 @@
+const ObjectID = require('bson-objectid').default;
+const logging = require('@tryghost/logging');
+
+const {createTransactionalMigration} = require('../../utils');
+
+module.exports = createTransactionalMigration(
+    async function up(knex) {
+        const members = await knex('members')
+            .select('id', 'created_at');
+
+        const toInsert = [];
+
+        // eslint-disable-next-line no-restricted-syntax
+        for (const member of members) {
```

Comment:
@daniellockyer Are you okay with this loop?

Reply:
Yeah it should be ok, but would it also be better to use `.map` here?

Reply:
Don't think it'll make a difference RE: perf, that's just a style thing AFAIC - what do you prefer?

Reply:
I've swapped it, `.map` does make sense here as it avoids all the pushes :)